test/python/transpiler/test_stochastic_swap.py

```diff
@@ -502,58 +429,30 @@ def test_congestion(self):
         #     |
         # 0 - 1 - 3
 
-        expected = QuantumCircuit(qr, ar, cr)
-        expected.cx(qr[1], ar[0])
-        expected.h(ar[0])
+        expected = QuantumCircuit(qr, cr)
+        expected.cx(qr[1], qr[2])
+        expected.h(qr[2])
         expected.swap(qr[0], qr[1])
         expected.h(qr[0])
-        expected.cx(qr[1], ar[1])
+        expected.cx(qr[1], qr[3])
         expected.measure(qr[1], cr[0])
-        expected.h(ar[1])
-        expected.swap(qr[1], ar[1])
-        expected.cx(ar[0], qr[1])
-        expected.h(ar[1])
-        expected.swap(qr[1], ar[1])
-        expected.measure(ar[0], cr[2])
-        expected.measure(ar[1], cr[3])
+        expected.h(qr[3])
+        expected.swap(qr[1], qr[3])
+        expected.cx(qr[2], qr[1])
+        expected.h(qr[3])
+        expected.swap(qr[1], qr[3])
+        expected.measure(qr[2], cr[2])
+        expected.measure(qr[3], cr[3])
         expected.cx(qr[1], qr[0])
         expected.measure(qr[1], cr[0])
         expected.measure(qr[0], cr[1])
         expected_dag = circuit_to_dag(expected)
 
-        layout = Layout({qr[0]: 0, qr[1]: 1, ar[0]: 2, ar[1]: 3})
-
-        pass_ = StochasticSwap(coupling, layout, 20, 13)
+        pass_ = StochasticSwap(coupling, 20, 13)
         after = pass_.run(dag)
 
         self.assertEqual(expected_dag, after)
 
-    def test_all_single_qubit(self):
```

Comment:
Is there another "pass is a nop if circuit contains only single qubit gates" test? If not, would this be worth keeping, just on a single register?

Reply:
i restored this test.